src/main/java/com/opentable/db/postgres/embedded/ProcessOutputLogger.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.opentable.db.postgres.embedded;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+
+import org.slf4j.Logger;
+
+/**
+ * Read standard output of process and write lines to given {@link Logger} as INFO;
+ * depends on {@link ProcessBuilder#redirectErrorStream(boolean)} being set to {@code true} (since only stdout is
+ * read).
+ */
+final class ProcessOutputLogger implements Runnable {
+    @SuppressWarnings("PMD.LoggerIsNotStaticFinal")
+    private final Logger logger;
+    private final Process process;
+    private final BufferedReader reader;
+
+    private ProcessOutputLogger(final Logger logger, final Process process) {
+        this.logger = logger;
+        this.process = process;
+        reader = new BufferedReader(new InputStreamReader(process.getInputStream(), StandardCharsets.UTF_8));
```

Comment:
I guess the underlying inputstream from the process is cleaned up earlier?

Reply:
Ooh, good catch sir.  Implemented in a60463892c7892d09562faf8b9cdeb73e4214913...69841557b0a2dae9ffaa10d52db9a0cbf46eaab7.

We always perform the error → output redirection, so we don't need to do this for the error stream.